packages/components/README.md

```diff
@@ -1,3 +1,89 @@
+
+- [@ovh-ux/ui-kit.css](https://github.com/ovh/ovh-ui-kit/blob/master/packages/css/README)
+- [angular](https://www.npmjs.com/package/angular)
+- [angular-aria](https://www.npmjs.com/package/angular-aria)
+- [angular-sanitize](https://www.npmjs.com/package/angular-sanitize)
+
+You need to load these optional dependencies in your project depending on component used:
+
+- [bloodhound-js](https://www.npmjs.com/package/bloodhound-js) (used by [Autocomplete](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/autocomplete/README))
+- [clipboard](https://www.npmjs.com/package/clipboard) (used by [Clipboard](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/clipboard/README))
+- [flatpickr](https://www.npmjs.com/package/flatpickr) (used by [Calendar](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/calendar/README) and [Timepicker](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/timepicker/README))
+- [popper.js](https://www.npmjs.com/package/popper.js) (used by 
+[Popover](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/popover/README) and 
+[Tooltip](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/tooltip/README) and
+[Dropdown](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/dropdown/README) and 
+[Autocomplete](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/autocomplete/README))
+- [ui-select](https://www.npmjs.com/package/ui-select) (used by [Select](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/select/README) and [Select Picker](https://github.com/ovh/ovh-ui-kit/blob/master/packages/components/select-picker/README))
+
+### Install with [npm](https://www.npmjs.com/)
+
+```bash
+npm install @ovh-ux/ui-kit.components
+```
+
+### Install with [yarn](https://yarnpkg.com)
+
+```bash
+yarn add @ovh-ux/ui-kit.components
+```
+
+## Usage
+
+1. In your `index.html`, you need to load the component and its dependencies.
+
+```html
+<link rel="stylesheet" href="path/to/@ovh-ux/ui-kit.css/dist/css/oui.css">
+
+<script src="path/to/@ovh-ux/ui-kit.components/dist/js/oui-{componentName}.js" type="text/javascript"></script>
+```
+
+2. You need to add `{componentName}` in your angular module dependencies like:
+
+```js
+angular.module('myAwesomeApp', ['{componentName}']);
+```
+
+Note: Replace {componentName} with actual component like 'oui-button' for button component.
+
+Refer to component README file for more details.
+
+### Webpack
+
+In your `index.js`, you need to import the component, with CSS and other dependencies separately.
+
+```js
+import '@ovh-ux/ui-kit.css/dist/css/oui.css';
+import ouiComponent from  '@ovh-ux/ui-kit.components/dist/js/oui-{componentName}.js';
+angular.module('myAwesomeApp', ['ouiComponent']);
+```
+
+Note: Replace {componentName} with actual component like 'button' for button component.
+
+Refer to component README file for more details.
+
+## Documentation
```

Comment:
You could also mention the Storybook documentation url: https://ovh.github.io/ovh-ui-kit/